api/src/services/users.ts

```diff
@@ -339,7 +339,7 @@ export class UsersService extends ItemsService {
 			throw new InvalidPayloadException(`Email address ${email} hasn't been invited.`);
 		}
 
-		const passwordHashed = await argon2.hash(password);
+		const passwordHashed = generateHash(password);
```

Comment:
Bob Saget! I forgot an `await` here. :hurtrealbad:

Reply:
Happens to the best of us! It was already reported, and then fixed by @aidenfoxx ☺️